swf/settings.py

```diff
@@ -105,13 +105,16 @@ def from_env():
     :rtype: dict
 
     """
-    try:
-        return {'aws_access_key_id': os.environ['AWS_ACCESS_KEY_ID'],
-                'aws_secret_access_key': os.environ.get('AWS_SECRET_ACCESS_KEY'),
-                'region': os.environ.get('AWS_DEFAULT_REGION')
-        }
-    except KeyError:
-        return {}
+    hsh = {}
+
+    if "AWS_ACCESS_KEY_ID" in os.environ:
```

Comment:
Is this really needed? Don't we use boto, which is perfectly able to fetch these data from all the standard places?
(If not, :+1: obviously)